cpp/include/cudf/functions.h

```diff
@@ -384,62 +384,17 @@ gdf_error gdf_hash_partition(int num_input_cols,
 /** 
  * @Synopsis  Computes the prefix sum of a column
  * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_generic(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_i8(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
+ * @Param[in] input Input column for prefix sum with null_count = 0
+ * @Param[out] output The output column containing the prefix sum of the input
```

Comment:
Make sure to document that the output column's buffers are expected to already be allocated.